src/main.jsx

```diff
@@ -43,6 +45,14 @@ const router = createBrowserRouter([
         path: "login",
         element: <LoginPage />,
       },
+      {
+        path: "ccm",
+        element: <CCMPage />,
+      },
+      {
+        path: "cto",
+        element: <CTOPage />,
+      },
```

Comment:
please change your path to the complete name

Reply:
Path has been updated to the complete name